src/index.js

```diff
@@ -0,0 +1,56 @@
+import './style.css';
+
+const list = document.createElement('ul');
+const container = document.querySelector('.container');
+const header = document.createElement('h1');
+const btn = document.createElement('btn');
+
+btn.innerHTML = 'All completed';
+header.innerHTML = "Today's todo";
+header.classList = 'header';
+list.classList = 'list';
+
+const input = document.createElement('input');
+input.placeholder = 'Add to your list';
+const todo = [
+  {
+    description: 'reading books',
+    completed: true,
+    index: Number,
+  },
```

Comment:
- [ ] [OPTIONAL] If you implement like this structure below it would be better because in this milestone you are preparing a structure for the next milestone so your structure should be clear, it would be good if you set `Number` to `123..` and `boolean` to false because they are not yet complete:

```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```